spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'foods index page' do
+  context 'you want to search for foods by ingredient' do
+    it 'root search redirects to foods path' do
+      visit root_path
+      fill_in :q, with: 'sweet potatoes'
+      click_on :commit
+      
+      expect(current_path).to eq(foods_path)
+    end
+  end
+  
+  context 'you have searched sweet potatoes' do
+    before :each do
+      visit root_path
+      fill_in :q, with: 'sweet potatoes'
+      click_on :commit
+    end
```

Comment:
These could be combined. Really the top it block should be within the context block you begin on 14. Again, the API call should be stubbed out.